rskj-core/src/main/java/org/ethereum/config/blockchain/HardForkActivationConfig.java

```diff
@@ -20,18 +20,17 @@
 import com.typesafe.config.Config;
 
 public class HardForkActivationConfig {
-    private final int firstFork;
+    private final int orchid;
```

Comment:
Maybe `orchidActivationHeight`?

Reply:
Agree, not very clear without reading the rest of the code.